Player.tscn

```diff
@@ -9,5 +41,29 @@
 [node name="PlayerCamera" type="ARVRCamera" parent="PlayerOrigin"]
 transform = Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1.85, 0 )
 
+[node name="LeftHandController" type="ARVRController" parent="PlayerOrigin"]
+transform = Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, -0.1, 0, 0 )
+
+[node name="LeftHandMesh" type="MeshInstance" parent="PlayerOrigin/LeftHandController"]
+transform = Transform( -1, 0, -8.74228e-008, 0, 1, 0, 8.74228e-008, 0, -1, 0, 0, 0 )
+mesh = SubResource( 1 )
+material/0 = null
+
+[node name="CollisionShape" type="CollisionShape" parent="PlayerOrigin/LeftHandController"]
+transform = Transform( 0.2, 0, 0, 0, 0.2, 0, 0, 0, 0.2, 0, 0, -0.0223315 )
+shape = SubResource( 2 )
+
+[node name="RightHandController" type="ARVRController" parent="PlayerOrigin"]
```

Comment:
set `controller_id=2` for `RightHandController`

as stated here https://docs.godotengine.org/en/3.1/classes/class_arvrcontroller.html#property-descriptions
```A controller id of 0 is unbound and will always result in an inactive node. Controller id 1 is reserved for the first controller that identifies itself as the left hand controller and id 2 is reserved for the first controller that identifies itself as the right hand controller.```